userChrome.css

```diff
@@ -338,6 +339,26 @@
     border-radius: var(--tab-border-radius) var(--tab-border-radius) 0 0 !important; /* Original: var(--tab-border-radius) */
   }
 
+  /* Coner Rounding */
+  #tabs-newtab-button:hover {
+    /* Color */
+    fill: color-mix(in srgb, currentColor 17%, transparent) !important; /* Hardcoded compatibility var(--toolbarbutton-hover-background) */
+    -moz-context-properties: fill !important;
+
+    /* Corner Image */
+    --newtab-position: calc((var(--tab-corner-rounding) / 2) * -1);
```

Comment:
There should be no decimal point. This value is `2.5px`, so It is not symmetrical.
![image](https://user-images.githubusercontent.com/47362439/123595287-b19d4700-d82b-11eb-80ef-694e1cda835d.png)

Reply:
I got it.

Nox, I did a comparison.
`--tab-corner-rounding: 4px`
![image](https://user-images.githubusercontent.com/25581533/123599991-c0610b00-d7e5-11eb-9b51-7b90df42f0ac.png)

`--tab-corner-rounding: 6px`
![image](https://user-images.githubusercontent.com/25581533/123600104-dd95d980-d7e5-11eb-9224-951fe31b68c6.png)

tab => `4px`, newtab button => `6px`
![image](https://user-images.githubusercontent.com/25581533/123601667-7842e800-d7e7-11eb-834e-09d32deeabd9.png)

Personally, I like the last one.